### ISSUE TITLE
Cache Expiry and Persistence for "load_image_async" Function

### ISSUE THREAD

**Issue by munishkhatri720** [shahriyardx/easy-pil#36]
Does this will not increase memory usage of the application in which easy_pil is used ? 
![image](https://github.com/shahriyardx/easy-pil/assets/109850452/3b76a4b5-1650-4c1a-a243-dbf98f70622f)

I have an api which builds musiccard every second and I noticed my vps memory usage touched 1gb in just few minutes 
![image](https://github.com/shahriyardx/easy-pil/assets/109850452/9c75443d-d1e1-4f6b-a44a-6b67be05aa19)

**Comment by shahriyardx** [comment]
Are you closing the editor after generating images ?

**Comment by munishkhatri720** [comment]
Yes, I am closing the editor after modifying the image. But my question is, why is the cache TTL set to 12 hours? I used easy_pil with FastAPI, which handles 22,000 images every day, but the API's RAM usage reached 7 GB in a single day. I think the load_image_async function is caching images for too long, which is causing the high RAM usag

**Comment by shahriyardx** [comment]
So, do you think removing cache is better so you can implement your own caching strategy? 
Also `load_image` loads the image from a link then opens it using `Image.open()` you need to close this too after the usage of the image is done

```py
editor = Editor()

img = await load_image_async(url)
editor.paste(img, ...)

img.close()
bytes = editor.image_bytes
editor.close()
```

Let me know if this strategy helps reducing ram usage, ultimately all the images that you open must be closed to improve performance

**Comment by munishkhatri720** [comment]
> So, do you think removing cache is better so you can implement your own caching strategy? 
> Also `load_image` loads the image from a link then opens it using `Image.open()` you need to close this too after the usage of the image is done
> 
> ```py
> editor = Editor()
> 
> img = await load_image_async(url)
> editor.paste(img, ...)
> img.close()
> 
> bytes = editor.image_bytes
> editor.close()
> ```
> 
> Let me know if this strategy helps reducing ram usage, ultimately all the images that you open must be closed to improve performance
> 

Yes, remove the cache strategy and allow people to implement their own. Otherwise, many users won't see the source code and will start using it in production. Later, they may complain about memory leaks. Alternatively, you could implement an LFU-based caching system that caches up to a set limit.

**Comment by shahriyardx** [comment]
ok I have removed the caching in version 0.4.0
